Packages/vcs/Lib/VTKPlots.py

```diff
@@ -458,6 +465,7 @@ def isopened(self):
 
     def geometry(self, x, y, *args):
         self.renWin.SetSize(x, y)
+        self._lastSize = (x, y)
```

Comment:
I'm not sure but since we do it everywhere else where the size changes I added it as well. @chaosphere2112 do we still need this when we switch to your GUI